src/Resources.js

```diff
@@ -38,8 +38,21 @@ const Resources = () => {
       return;
     }
 
-    const json = await getResources(tenant, site);
-    setResources(json);
+    var finalJson = [];
+    if(Array.isArray(site)){
+      for(const s of site){
+        const json = await getResources(tenant, s);
+        finalJson = finalJson.concat(json);
+      }
+    } else {
+      finalJson = await getResources(tenant, site);
+    }
+    finalJson.forEach (item => {
+      //set a unique key with <tenant id>::<site id>::<resource id>
+      item["Pkey"] = item["TenantID"] + "-" + item["SiteID"] + "-" + item["ResourceID"];
```

Comment:
So my gut reaction is wondering if this can be simplified and made more "javascript"-y. 

instead of building a unique key, you could just build a an object with embedded objects so you could something instead of:

item["pkey"] = "tenant1-site1-resource1"

Just create an overall object that is something like:

finalJson["tenant1"]["site1"]["resource1"] 

and pass that to the resourceGrid. It'll be easier to parse for that way imo.

Reply:
I went by eg https://www.telerik.com/kendo-react-ui/components/grid/selection/. For the select to work we needed an unique key.